tests/typing_test.py

```diff
@@ -0,0 +1,45 @@
+from honeybee_radiance_command._typing import path_checker, path_checker_multiple
+import pytest
+import os
+
+# TODO: Add tests for all methods in the typing module.
+
+def test_path_checker():
+    test_path1 = r'test/some directory/some_file.rad'
+    test_path2 = r'test/some directory/some_file.bmp'
+
+    pth1 = path_checker(test_path1)
+    if os.name == 'nt':
+        assert pth1 == r'"test\some directory\some_file.rad"'
+    else:
+        assert pth1 == r"'test/some directory/some_file.rad'"
+
+    with pytest.raises(ValueError):
+        pth2 = path_checker(test_path2, extn_list=['.rad', '.sky'])
+
+
+def test_path_checker_multiple():
+    test_path1 = r'test/some directory/some_file.rad'
+    test_path2 = r'test/some directory/some_file.bmp'
+    test_path3 = r'test/some directory/some_file2.rad'
+
+    pth_list = path_checker_multiple([test_path1, test_path2])
+    if os.name == 'nt':
+        assert pth_list == [r'"test\some directory\some_file.rad"',
+                            r'"test\some directory\some_file.bmp"']
+    else:
+        assert pth_list == [r"'test/some directory/some_file.rad'",
+                            r"'test/some directory/some_file.bmp'"]
+
+    pth_list_str = path_checker_multiple([test_path1, test_path2],
+                                         outputs_as_string=True)
+    if os.name == 'nt':
+        assert pth_list_str == r'"test\some directory\some_file.rad" ' \
+                               r'"test\some directory\some_file.bmp"'
+    else:
+        assert pth_list_str == r"'test/some directory/some_file.rad' " \
+                               r"'test/some directory/some_file.bmp'"
```

Comment:
As I commented above this should always return a unix-like path.

Reply:
It should but it won't because of what I pointed out here: https://github.com/ladybug-tools/honeybee-radiance-command/pull/209#discussion_r762013118. 
I think we should then fix/update normpath first. I can add that posix change there... I am fairly certain that it wont break anything as all of these methods/classes are talking to Radiance only.

Reply:
> I am fairly certain that it wont break anything as all of these methods/classes are talking to Radiance only

I wish that was how the world works! 😄 I have to refer you to this quote [here](https://clig.dev/#robustness-guidelines): 

![image](https://user-images.githubusercontent.com/2915573/144625567-e0a6bb51-de03-4b79-8a1d-a1d94ee10ccb.png)

Reply:
@mostaphaRoudsari Okay, well then I am somewhat certain that it wont break anything :P !